spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -39,6 +44,32 @@
       expect(json['longitude'].to_f).to eq(new_target.longitude.to_f)
       expect(json['topic_id']).to eq(new_target.topic_id)
       expect(json['user_id']).to eq(new_target.user_id)
+      expect(json['match_targets']).to be_empty
+    end
+
+    context 'with matched target' do
+      it 'returns target information with matched target' do
+        params[:target][:latitude] = target.latitude + 0.0001
+        params[:target][:longitude] = target.longitude + 0.0001
+        params[:target][:topic_id] = target.topic_id
+        subject
+        new_target = Target.find_by_title(params[:target][:title])
+        expect(json).not_to be_empty
```

Comment:
This `expect` is unnecessary as the next ones include testing that the response is not empty